packages/terra-site/src/examples/content-container/ContentContainerFill.jsx

```diff
@@ -23,10 +24,11 @@ pharetra, augue felis aliquet nunc, non facilisis est velit vel elit.
                  </p>);
 
 const contentheader = <Heading level={1} size={'large'}>Display Header</Heading>;
+const contentfooter = <Button text="Save" />;
```

Comment:
Nit: For consistency and clarity in the examples, can we rename the footer text to 'Display Footer'?

Reply:
@yuderekyu Sure, should I use `Heading` as the content header does instead of `Button`?

Reply:
I think the button is fine, just specifying that it's the footer element is clearer to me.